datacosmos/datacosmos.py

```diff
@@ -0,0 +1,159 @@
+"""Contains the main entry point for the DataCosmos API SDK."""
+
+from __future__ import annotations
+
+import datetime
+import os
+from typing import Dict, List
+
+import geojson
+
+from datacosmos.auth import DataCosmosCredentials
+from datacosmos.const import (
+    DATACOSMOS_PRODUCTION_BASE_URL,
+    Constellations,
+    Levels,
+    Satellites,
+)
+from datacosmos.stac import search as stac_search
+
+
+class DataCosmos:
+    """This is the main entry point for the DataCosmos API SDK.
+
+    You can use this to search for items in the DataCosmos catalogue and then
+    download them.
+
+    The SDK can be initialised in two ways:
+
+    1. With credentials loaded from environment variables
+    2. With credentials loaded from a file
+
+    With credentials from environment variables DATACOSMOS_KEY_ID and
+    DATACOSMOS_KEY_SECRET:
+
+    .. code-block:: python
```

Comment:
Is this a standard, supported universally, only supported by certain editors/etc...?

Reply:
It's a sphinx thing, muscle memory, I'll get rid of it since we probably won't be using sphinx for docs

Reply:
updated to use `>>>` which is widely used in python docs and gives highlighting in most editors